external/java/src/main/java/ch/epfl/dedis/byzcoin/contracts/DarcInstance.java

```diff
@@ -205,10 +221,31 @@ public Proof spawnContractAndWait(String contractID, Signer s, List<Argument> ar
                 throw new CothorityCommunicationException("this is not a correct darc-spawn");
             }
         }
-        logger.info("waiting on iid {}", iid);
         return bc.getProof(iid);
     }
 
+    /**
+     * Spawns a new darc from the given darcInstance. The current darc needs to have the "spawn:darc"
+     * rule with an expression that is resolvable with the signer s.
```

Comment:
"resolvable" might not be the right word, perhaps "...with an expression that can be evaluated to true using signer s"?